internal/mq/msgstream/mqwrapper/kafka/kafka_consumer.go

```diff
@@ -219,6 +220,33 @@ func (kc *Consumer) GetLatestMsgID() (mqwrapper.MessageID, error) {
 	return &kafkaID{messageID: high}, nil
 }
 
+func (kc *Consumer) CheckPreTopicValid(topic string) error {
+	latestMsgID, err := kc.GetLatestMsgID()
+	// check topic is existed
+	if err != nil {
+		switch v := err.(type) {
```

Comment:
use `errors.Is()` to check error, to make sure you are always able to get the concerned error